components/base/wheeled/kinematics.go

```diff
@@ -23,13 +20,21 @@ type kinematicWheeledBase struct {
 
 // WrapWithKinematics takes a wheeledBase component and adds a slam service to it
 // It also adds kinematic model so that it can be controlled.
-func WrapWithKinematics(ctx context.Context, base *wheeledBase, slam slam.Service) (base.KinematicBase, error) {
+func (base *wheeledBase) WrapWithKinematics(ctx context.Context, slamSvc slam.Service) (base.KinematicBase, error) {
```

Comment:
[q] Does this function need to be public anymore?

Reply:
Yes, as it will be used by the motionplan package in a subsequent PR that relies on this work